packages/components/modal/src/ModalHeader/ModalHeader.tsx

```diff
@@ -40,6 +42,11 @@ export const ModalHeader = ({
     >
       <Subheading as="h2" isTruncated marginBottom="none">
         {title}
+        {subtitle && (
+          <Text marginLeft="spacingXs" color="grey700">
```

Comment:
```suggestion
          <Text marginLeft="spacingXs" color="gray700">
```
I also do the same mistake every time, hate that we have this difference with design tokens 🤦‍♂️

@damann @fabe will it be too much work/problematic to rename it in designs?